src/news/newsContainer.ts

```diff
@@ -4,7 +4,8 @@ import { GlobalState } from 'types'
 import News from './news'
 
 export const mapStateToProps = (state: GlobalState) => ({
-  newsItems: state.news
+  newsItems: state.news.newsItems,
```

Comment:
Don't know if this would be clearer to look at or not but I think you can just do destructuring i.e. `{ ...state.news }`